src/OrchardCore/OrchardCore.DisplayManagement/Descriptors/ShapeTable.cs

```diff
@@ -94,7 +94,6 @@ public bool TryGetValue(string shapeAlternate, out ShapeBinding binding)
                 {
                     binding = new ShapeBinding
                     {
-                        ShapeDescriptor = descriptor,
```

Comment:
I remember when I added it, it was to be able to find the source of a shape when I developed the Shape Tracing tool. I feel better we get rid of it to be honest ...